### ISSUE TITLE
No docs.rs for fs4 0.9.0

### ISSUE THREAD

**Issue by mstange** [al8n/fs4-rs#25]
https://docs.rs/crate/fs4/latest is currently saying "docs.rs failed to build fs4-0.9.0"

Here's the failure log from https://docs.rs/crate/fs4/0.9.0/builds/1305151 :

```
# rustc version
rustc 1.82.0-nightly (7c2012d0e 2024-07-26)# docs.rs version
docsrs 0.6.0 (bd66f012 2024-07-12)# build log
[INFO] running `Command { std: "docker" "create" "-v" "/home/cratesfyi/workspace-builder/builds/fs4-0.9.0/target:/opt/rustwide/target:rw,Z" "-v" "/home/cratesfyi/workspace-builder/builds/fs4-0.9.0/source:/opt/rustwide/workdir:ro,Z" "-v" "/home/cratesfyi/workspace-builder/cargo-home:/opt/rustwide/cargo-home:ro,Z" "-v" "/home/cratesfyi/workspace-builder/rustup-home:/opt/rustwide/rustup-home:ro,Z" "-e" "SOURCE_DIR=/opt/rustwide/workdir" "-e" "CARGO_TARGET_DIR=/opt/rustwide/target" "-e" "DOCS_RS=1" "-e" "CARGO_HOME=/opt/rustwide/cargo-home" "-e" "RUSTUP_HOME=/opt/rustwide/rustup-home" "-w" "/opt/rustwide/workdir" "-m" "6442450944" "--cpus" "6" "--user" "1001:1001" "--network" "none" "ghcr.io/rust-lang/crates-build-env/linux@sha256:dff56e7819e73ed36160586b3445e93eb0be776c16704aeeded9c3fb668b2384" "/opt/rustwide/cargo-home/bin/cargo" "+nightly" "rustdoc" "--lib" "-Zrustdoc-map" "--all-features" "--config" "build.rustdocflags=[\"--cfg\", \"docsrs\", \"--cfg\", \"docsrs\", \"-Z\", \"unstable-options\", \"--emit=invocation-specific\", \"--resource-suffix\", \"-20240726-1.82.0-nightly-7c2012d0e\", \"--static-root-path\", \"/-/rustdoc.static/\", \"--cap-lints\", \"warn\", \"--extern-html-root-takes-precedence\"]" "--offline" "-Zunstable-options" "--config=doc.extern-map.registries.crates-io=\"https://docs.rs/{pkg_name}/{version}/x86_64-unknown-linux-gnu\"" "-Zrustdoc-scrape-examples" "-j6" "--target" "x86_64-unknown-linux-gnu", kill_on_drop: false }`
[INFO] [stderr] WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
[INFO] [stdout] e60aad65bebaadf9d5a65c90edf2f0d34a24b7a78a0a89379c94587ecf3d4712
[INFO] running `Command { std: "docker" "start" "-a" "e60aad65bebaadf9d5a65c90edf2f0d34a24b7a78a0a89379c94587ecf3d4712", kill_on_drop: false }`
[INFO] [stderr] warning: target filter specified, but no targets matched; this is a no-op
[INFO] [stderr]  Documenting fs4 v0.9.0 (/opt/rustwide/workdir)
[INFO] [stderr] error: unexpected literal
[INFO] [stderr]   --> src/lib.rs:87:104
[INFO] [stderr]    |
[INFO] [stderr] 87 |               #[cfg_attr(docsrs, doc(cfg(any(feature = "smol", feature = "async-std", feature = "tokio", "fs-err-tokio"))))]
[INFO] [stderr]    |                                                                                                          ^^^^^^^^^^^^^^
[INFO] [stderr]    |
[INFO] [stderr]   ::: src/file_ext.rs:5:1
[INFO] [stderr]    |
[INFO] [stderr] 5  | / cfg_async!(
[INFO] [stderr] 6  | |     pub(crate) mod async_impl;
[INFO] [stderr] 7  | | );
[INFO] [stderr]    | |_- in this macro invocation
[INFO] [stderr]    |
[INFO] [stderr]    = note: this error originates in the macro `cfg_async` (in Nightly builds, run with -Z macro-backtrace for more info)
[INFO] [stderr] 
[INFO] [stderr] error: could not document `fs4`
[INFO] running `Command { std: "docker" "inspect" "e60aad65bebaadf9d5a65c90edf2f0d34a24b7a78a0a89379c94587ecf3d4712", kill_on_drop: false }`
[INFO] running `Command { std: "docker" "rm" "-f" "e60aad65bebaadf9d5a65c90edf2f0d34a24b7a78a0a89379c94587ecf3d4712", kill_on_drop: false }`
[INFO] [stdout] e60aad65bebaadf9d5a65c90edf2f0d34a24b7a78a0a89379c94587ecf3d4712
```

**Comment by al8n** [comment]
Thanks! Fix in `0.9.1`.

**Comment by mstange** [comment]
Thank you!
